tests/libs/datasets/new_cases_and_deaths_test.py

```diff
@@ -154,6 +154,38 @@ def test_spread_first_reported_value():
     pd.testing.assert_series_equal(expected, results)
 
 
+def test_spread_first_reported_value_2():
+    series = test_helpers.series_with_date_index([None, None, 10, 5, 0, 0, 0, 12, 0, 4])
+
+    results = new_cases_and_deaths.spread_first_reported_value_after_stall(series)
+
+    expected = test_helpers.series_with_date_index([None, None, 10, 5, 3, 3, 3, 3, 2, 2])
+
+    pd.testing.assert_series_equal(expected, results)
```

Comment:
Just added this test because I wanted to make sure it handled multiple sequences of 0's right...

Reply:
Is this testing something other than https://github.com/covid-projections/covid-data-model/pull/1157/files#diff-3a62181b13755a395bfa139d8cc2f19ba2143e147a614ed4fb8adaf1df77972fR201?

Reply:
Oh, I'm not sure that link works, but meant to point at the `test_spreading_multiple_zeros` at the bottom

Reply:
Oops, no, haha.  Feel free to remove.